blueocean-core-js/src/js/capability/CapabilityAugmenter.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Created by cmeyers on 9/8/16.
+ */
+
+const addClass = (clazz, classMap) => {
+    const className = clazz._class;
+
+    if (!classMap[className]) {
+        // eslint-disable-next-line no-param-reassign
+        classMap[className] = [];
+    }
+
+    classMap[className].push(clazz);
+};
+
+const canWalk = (item) => item && (typeof item === 'object' || Array.isArray(item));
+
+// TODO: can we really filter out 'actions' in all cases? should it use a leading underscore?
+const IGNORED_PROPS = ['_links', 'actions'];
```

Comment:
@scherler I will add an option to fetch data for "actions" too. It's very large, so that's why I was hoping we could disable it. Which REST API calls are you examining the actions for?